desktop/core/src/desktop/management/commands/runruff.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+# Licensed to Cloudera, Inc. under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  Cloudera, Inc. licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os.path
+import subprocess
+import sys
+import argparse
+
+from django.core.management.base import BaseCommand
+from django.utils.translation import gettext as _
+
+from desktop.lib import paths
+
+
+LOG = logging.getLogger()
+
+
+class Command(BaseCommand):
+  help = _("""
+    Run Ruff code quality checks on Python files.
+
+    This command runs Ruff on a set of Python files, either specified explicitly or
+    determined by comparing the current branch with a specified diff branch.
+
+    Options:
+      --diff-branch=<branch>  Compare the current branch with the specified branch
+                              and check only the files that have changed.
+      All original Ruff commands are supported, including --fix to automatically fix errors.
+
+    Examples:
+      Run Ruff linting on specific Python files:
+        ./build/env/bin/hue runruff check file1.py file2.py
+
+      Run Ruff linting on files changed between the current branch and a specified diff branch:
+        ./build/env/bin/hue runruff check --diff-branch=origin/master
```

Comment:
IIUC we don't need to provide the --diff-branch arg always if we want to check with mater branch, right?
`./build/env/bin/hue runruff check` this cmd will do the job?